### ISSUE TITLE
Test failure using panda

### ISSUE THREAD

**Issue by dnebauer** [perl6/Perl6-MIME-Base64#8]
```
==> Fetching MIME::Base64
==> Building MIME::Base64
==> Testing MIME::Base64
This type does not support elems
  in any  at gen/moar/m-Metamodel.nqp line 1723
  in method decode at /path/to/.panda-work/1457223424_1/lib/MIME/Base64/Perl.pm6 line 103
  in method decode-str at /path/to/.panda-work/1457223424_1/lib/MIME/Base64.pm6 line 10
  in method decode_base64 at /path/to/.panda-work/1457223424_1/lib/MIME/Base64.pm6 line 19
  in block <unit> at t/basic.t line 14

# Looks like you planned 18 tests, but ran 1
t/basic.t ................. 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 17/18 subtests 
t/binary-and-long-line.t .. ok
t/oneline.t ............... ok
This type does not support elems
  in any  at gen/moar/m-Metamodel.nqp line 1723
  in method decode at /path/to/.panda-work/1457223424_1/lib/MIME/Base64/Perl.pm6 line 103
  in method decode-str at /path/to/.panda-work/1457223424_1/lib/MIME/Base64.pm6 line 10
  in block <unit> at t/rfc4648-test-vector.t line 23

# Looks like you planned 16 tests, but ran 7
t/rfc4648-test-vector.t ... 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 9/16 subtests 

Test Summary Report
-------------------
t/basic.t               (Wstat: 65280 Tests: 1 Failed: 0)
  Non-zero exit status: 255
  Parse errors: Bad plan.  You planned 18 tests but ran 1.
t/rfc4648-test-vector.t (Wstat: 65280 Tests: 7 Failed: 0)
  Non-zero exit status: 255
  Parse errors: Bad plan.  You planned 16 tests but ran 7.
Files=4, Tests=21,  4 wallclock secs ( 0.02 usr  0.00 sys +  3.42 cusr  0.18 csys =  3.62 CPU)
Result: FAIL
```

**Comment by ronaldxs** [comment]
"This type does not support elems" error also produced by:
```Perl6
my @a; my $b = Buf.new(@a)
```
which is what is happening inside MIME::Base64::Perl
Also discussed on [IRC #perl6 ...](http://irclog.perlgeek.de/perl6/2016-03-06#i_12144431)

Several possible workarounds including:
```Perl6
my @a = (); my $b = Buf.new(@a)
```
which was used in workaround pull request.

**Comment by zakame** [comment]
Hi all,

[zef](https://github.com/ugexe/zef) seems to be now the preferred module manager for Perl 6, while [panda](https://github.com/tadzik/panda) is now deprecated.  Let me close this issue, moving forward.

**Comment by jonathanstowe** [comment]
It appears it was fixed in the 908be4eb73d77b6c1fae20815836371f5028fcd3 (wasn't actually anything to do with panda.)
